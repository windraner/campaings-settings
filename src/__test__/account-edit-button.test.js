import React from 'react';
import AccountEditButton from '../components/main-content-section/tab-block/common/account-edit-button/account-edit-button';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

describe('<AccountEditButton />', () => {
  it('matches the snapshot hide', () => {
    const tree = shallow(<AccountEditButton />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
