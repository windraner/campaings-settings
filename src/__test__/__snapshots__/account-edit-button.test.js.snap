// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AccountEditButton /> matches the snapshot hide 1`] = `
<a
  className="campaigns-settings__account-edit-button d-flex align-items-end text-center"
  href="#"
>
  <span
    className="campaigns-settings__account-edit-button-icon mr-2"
  />
  Edit accounts
</a>
`;
